src/content/docs/release-notes/docs-release-notes/docs-4-12-2024.mdx

```diff
@@ -0,0 +1,51 @@
+---
+subject: Docs
+releaseDate: '2024-04-12'
+version: '4-12-2024'
+---
+
+<Callout variant="important">
+We're trying out release notes to let you know what's changing in our docs each week. This is an experiment, so [let us know via GitHub issue](https://github.com/newrelic/docs-website/issues/new/choose) if you find them helpful!
+</Callout>
+
+### New docs
+
+* General availability for browser [Session replay](/docs/browser/browser-monitoring/browser-pro-features/session-replay/), previously in limited preview. Check it out for deeper insights into user interactions on your websites.
```

Comment:
"browser Session replay" feels like odd capitalization to me...

"browser monitoring's Session replay" feels a little less off to me